dynamic_rest/fields/fields.py

```diff
@@ -118,8 +122,15 @@ def bind(self, *args, **kwargs):
         parent_model = getattr(self.parent.Meta, 'model', None)
 
         remote = is_field_remote(parent_model, self.source)
+
+        def get_model_field():
```

Comment:
I saw that, but there's a whole lot of logic I don't quite follow in that helper function. (See the "except" business there). Is that all correct here? I'm not sure, so I thought it safer to use this simpler logic that's identical with previous. Thoughts?